common/src/main/scala/uk/ac/wellcome/elasticsearch/mappings/RecordsIndex.scala

```diff
@@ -0,0 +1,26 @@
+package uk.ac.wellcome.elasticsearch.mappings
+
+import com.sksamuel.elastic4s.ElasticDsl._
+import com.sksamuel.elastic4s._
+import com.sksamuel.elastic4s.analyzers._
+import com.sksamuel.elastic4s.mappings.DynamicMapping
+
+class RecordsIndex(client: ElasticClient, indexName: String) {
```

Comment:
We should test at the very least that this creates an index appropriately named.